rackcity/views/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .it_model_views import model_detail, model_list
+from .it_model_views import model_detail, model_list, model_add
```

Comment:
Is it possible to do something like:
```
from .it_model_views import *
```

Reply:
technically yes, but I think it's general best practice to avoid wildcard imports (even if we are importing everything from that module anyway)